common/src/main/java/org/jboss/hal/testsuite/page/config/DeploymentScannerPage.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.hal.testsuite.cli.Library;
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.finder.FinderNavigation;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.formeditor.Editor;
+import org.jboss.hal.testsuite.util.Console;
+import org.jboss.hal.testsuite.util.PropUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Created by pcyprian on 22.10.15.
+ */
+public class DeploymentScannerPage extends ConfigurationPage {
+
+    public void navigateToDeploymentScanners() {
+        FinderNavigation navigation = new FinderNavigation(browser, StandaloneConfigEntryPoint.class)
+                .addAddress(FinderNames.CONFIGURATION, FinderNames.SUBSYSTEMS)
+                .addAddress(FinderNames.SUBSYSTEM, "Deployment Scanners");
+
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+    }
+
+    public ConfigFragment getWindowFragment() {
+        WebElement editPanel = browser.findElement(By.className("default-window-content"));
+        return  Graphene.createPageFragment(ConfigFragment.class, editPanel);
+    }
+
+    public Editor edit() {
```

Comment:
This method should delagate to related  `ConfigFragment` instead of copying `ConfigFragment.edit()` code. This will require one more private method - something like `getMainConfigFragment()`.